app/components/CategoryHeader.android.js

```diff
@@ -10,13 +11,33 @@ const styles = StyleSheet.create({
   },
 });
 
-const CategoryHeader = ({ category }) => (
-  <ToolbarAndroid
-    title={category.name}
-    style={styles.toolbar}
-    titleColor="white"
-  />
-);
+class CategoryHeader extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      icon: null
+    }
+  }
+
+  componentWillMount() {
```

Comment:
it looks like i can't:
`Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op.`